core/cowdata.h

```diff
@@ -87,7 +87,10 @@ class CowData {
 #if defined(_add_overflow) && defined(_mul_overflow)
 		size_t o;
 		size_t p;
-		if (_mul_overflow(p_elements, sizeof(T), &o)) return false;
+		if (_mul_overflow(p_elements, sizeof(T), &o)) {
+			*out = 0;
+			return false;
+		}
```

Comment:
@hpvb Please review. This is to silence this warning:
```
./core/cowdata.h:269:47: warning: 'alloc_size' may be used uninitialized in this function [-Wmaybe-uninitialized]
```
The alternative was to value-initialize `alloc_size`, but I guess this variant is more optimized.